### ISSUE TITLE
Adding Contributor Guidelines

### ISSUE THREAD

**Issue by kailashchoudhary11** [genicsblog/theme-files#64]
Creating contributor guidelines can help in maintaining the quality of contributions!

**Comment by gouravkhunger** [comment]
Hey @kailashchoudhary11, could you send a sample contributor guidelines for review before making PR? Thanks!

**Comment by Bennykillua** [comment]
@gouravkhunger Can I work on this?

**Comment by gouravkhunger** [comment]
sure @kailashchoudhary11 

@Bennykillua Sorry for not giving you this task, but could help me with [this issue](https://github.com/genicsblog/theme-files/issues/63)? I can help by showing you a demo on how to do the task and you could try to complete the rest 😄

**Comment by kailashchoudhary11** [comment]
@gouravkhunger You can refer to [this](https://github.com/github/docs/blob/main/CONTRIBUTING.md) for sample

**Comment by gouravkhunger** [comment]
Alright @kailashchoudhary11 that seems good. I think it would be better to update the main repo's [CONTRIBUTING.md](https://github.com/genicsblog/genicsblog.com/blob/main/CONTRIBUTING.md) as well, with the same content as this new one. That one doesn't look very good. It is also active at https://genicsblog.com/contribution-guide but isn't linked anywhere and is basically useless. Also it needs update.

**Comment by shockz09** [comment]
@gouravkhunger does this need any work?

**Comment by gouravkhunger** [comment]
@shockz09 Yes, but not a priority tbh. If you have a sample contributors guideline I can review and consider updating the current one.
